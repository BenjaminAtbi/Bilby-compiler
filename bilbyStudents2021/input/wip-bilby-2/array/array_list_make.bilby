main {
	imm a := [1,2,3];
	print a[2] $s a[1] $s a[0] $n;
	
	imm b := [2.1 as int];
	print b $n;
}