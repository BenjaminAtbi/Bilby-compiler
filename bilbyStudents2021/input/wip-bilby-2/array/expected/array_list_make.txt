Couldn't read file "D:\Projects\Bilby-compiler\bilbyStudents2021\input\wip-bilby-2/array/output/array_list_make.asm"
